src/main/java/org/folio/processing/events/services/publisher/KafkaEventPublisher.java

```diff
@@ -100,4 +103,17 @@ public KafkaEventPublisher(KafkaConfig kafkaConfig, Vertx vertx, int maxDistribu
     }
     return future;
   }
+
+  private void checkAndAddHeaders(String recordId, String chunkId, String jobExecutionId, List<KafkaHeader> headers) {
+    if (recordId == null) {
+      LOGGER.warn("RecordId is empty for jobExecutionId: '{}' ", jobExecutionId);
+    } else {
+      headers.add(KafkaHeader.header(RECORD_ID_HEADER, recordId));
+    }
+    if (chunkId == null) {
+      LOGGER.warn("ChunkId is empty for jobExecutionId: '{}' ", chunkId);
```

Comment:
There should be jobExecutionId instead of chunkId

Reply:
@AivarIUsupov thanks!